semgrep-core/src/core/Rule.ml

```diff
@@ -167,15 +167,30 @@ type pformula = New of formula | Old of formula_old [@@deriving show, eq]
 (* Taint-specific types *)
 (*****************************************************************************)
 
-type sanitizer_spec = {
+type taint_source = {
+  formula : pformula;
+  label : string;  (** The label to attach to the data *)
```

Comment:
Should it be an option? Or do you have a default value for it?
In that case maybe the default value should be defined in this file.